src/overlays/actors/ovl_En_Dg/z_en_dg.c

```diff
@@ -192,98 +238,106 @@ void func_80989204(EnDg* this, PlayState* play) {
     Actor_UpdateBgCheckInfo(play, &this->actor, 26.0f, 10.0f, 0.0f, 5);
 }
 
-void func_8098933C(EnDg* this, Vec3f* arg1) {
-    f32 sp24;
-    f32 sp20;
-    f32 sp1C;
+void EnDg_GetFloorRot(EnDg* this, Vec3f* floorRot) {
```

Comment:
We can decide on a standard name later. I called this function FloorTangent in other actors.